Dockerfile

```diff
@@ -1,14 +1,14 @@
 # Build App
-FROM openjdk:8-jdk-alpine AS BUILD
+FROM openjdk:11-jdk-slim AS BUILD
 
 ADD . /work
 WORKDIR /work
 RUN ./gradlew build --build-file ./build.gradle
```

Comment:
You can cache the Gradle wrapper using Docker layer cache. See https://medium.com/@nieldw/caching-gradle-binaries-in-a-docker-build-when-using-the-gradle-wrapper-277c17e7dd22 for more.